sale_order_deliver_status/models/sale_order.py

```diff
@@ -46,7 +46,7 @@ def _compute_delivery_status(self):
                     order.delivery_status = "delivered" if all_delivered else "partial"
 
     @api.multi
-    @api.depends("delivery_status", "invoice_status", "invoice_ids.state", "state")
+    @api.depends("delivery_status", "invoice_status", "state")
```

Comment:
@TimLai125 How should invoice.state be reflected back to order_status if the trigger is lost?